components/aws/sagemaker/unit_tests/tests/testUtils.py

```diff
@@ -0,0 +1,10 @@
+def check_empty_string_values(obj):
```

Comment:
This file name does not match that of our existing files. Python is typically snake_case for file names, so this would be `test_utils.py`. I understand this may be picked up by pytest, but either it won't run the file or there should be a way to configure it to ignore this file.

Reply:
done